tools/pipelines/test-azure-frs.yml

```diff
@@ -5,74 +5,148 @@
 
 name: $(Build.BuildId)
 
+schedules:
+  - cron: "0 4 * * *"
```

Comment:
Hey @ms-avocado, good question! I think it will depend on what kind of load we would expect these tests to add, and what FRS endpoints we would target. I'm not super up-to-date with the discussions around this area, so apologies if I am missing some important information. But it might be worth it having a separate conversation for us to align on this topic - is there someone else in the FRS team you have been working on regarding this? We can have a joint conversation to reach an agreement

Reply:
Sounds good, @ssimic2 is also on top of this topic, so would be good to include him.